includes/Core/Modules/Module.php

```diff
@@ -817,15 +821,29 @@ protected function exception_to_error( Exception $e, $datapoint ) {
 			if ( isset( $errors[0]['reason'] ) ) {
 				$reason = $errors[0]['reason'];
 			}
+		} elseif ( $e instanceof Google_Proxy_Code_Exception ) {
+			$status        = 401;
+			$auth_client   = $this->authentication->get_oauth_client();
+			$message       = $auth_client->get_error_message( $message );
+			$reconnect_url = $auth_client->get_proxy_setup_url(
+				$e->getAccessCode(),
+				$message
```

Comment:
This parameter is for the error code, which here technically is the `$message` variable _before_ being modified via `OAuth_Client::get_error_message`. I don't remember why we even need to pass the error code here, but we should make sure we pass a code and not a user-facing message.

Reply:
We don't use it directly, it is just sent to the `googlesitekit_proxy_setup_url_params` filter:

https://github.com/google/site-kit-wp/blob/0413b5fcc00e8773075edf9b4032487ee71e982f/includes/Core/Authentication/Clients/OAuth_Client.php#L821-L829

Anyway we can be confident here that we use error code because it is passed as error message here:

https://github.com/google/site-kit-wp/blob/0413b5fcc00e8773075edf9b4032487ee71e982f/includes/Core/Authentication/Clients/Google_Site_Kit_Proxy_Client.php#L142